yivi_core/lib/routing.dart

```diff
@@ -330,6 +347,39 @@ GoRouter createRouter(BuildContext buildContext, WidgetRef ref) {
                   );
                 },
               ),
+              GoRoute(
+                path: "/id_card",
+                builder: (context, state) {
+                  return PassportMrzManualEntryScreen(
```

Comment:
Maybe rename:
- `PassportMrzManualEntryScreen` to `DocumentMrzManualEntryScreen`
- `PassportMrzManualEntryTranslationKeys` to `DocumentMrzManualEntryTranslationKeys`
- `PassportNfcReadingRouteParams` to `DocumentNfcReadingRouteParams`